Rakefile

```diff
@@ -51,7 +51,15 @@ def getAgentTrackingInfo
   end
 end
 
+# Packaging into a tar
+require 'rake/packagetask'
+Rake::PackageTask.new("codedeploy-agent", :no_version) do |p|
```

Comment:
So to run this I would run `rake codedeploy-agent`? 

How about just calling the target `:tar` (colon in place of surrounding quotes)?

Reply:
Also I'd prefer empty string to `:no_version` -- the absence of a version implies no version.

Reply:
Would run `rake package` or `bundle exec rake package` to get the tar file. Looking into the tar file version naming.

Reply:
Used a hacky way so that the name of the file comes out as `codedeploy-agent.tar.gz`